src/libostree/ostree-repo.c

```diff
@@ -870,13 +870,30 @@ ostree_repo_get_boolean_option (OstreeRepo *self,
                                 gboolean   *result,
                                 GError    **error)
 {
+  g_autoptr(GError) temp_error = NULL;
+  gboolean ret_bool;
+
   g_return_val_if_fail (self->inited, FALSE);
 
-  if (!ot_keyfile_get_boolean_with_default (self->config, section,
-                                            key, default_value,
-                                            result, error))
-    return FALSE;
+  ret_bool = g_key_file_get_boolean (self->config, section, key, &temp_error);
+  if (temp_error != NULL)
+    {
+      if (g_error_matches (temp_error, G_KEY_FILE_ERROR, G_KEY_FILE_ERROR_KEY_NOT_FOUND))
+        {
+          if (self->parent_repo != NULL)
+            return ostree_repo_get_boolean_option (self->parent_repo,
```

Comment:
I'm uncertain about unilaterally inheriting *all* options from the parent repo.  For example, we obviously don't want `core/mode`.  This one wouldn't come in in practice since it'll always be set in a repo.

But another example is `core/fsync`.

Maybe the best example is `core/repo_version` (although interestingly that call doesn't use `ot_keyfile_get_boolean_with_default()` so it'd still work with this patch).

To flip this all around, are there *any* options we want to inherit other than the remotes?